src/test/java/pageobjects/SearchPage.java

```diff
@@ -0,0 +1,10 @@
+package pageobjects;
+
+import org.openqa.selenium.By;
+
+public class SearchPage {
+  public static final By ACCEPT_ALL = By.id("L2AGLb");
+  public static final By SEARCH_BUTTON = By.id("APjFqb");
+  public static final By FEELING_LUCKY = By.xpath("/html/body/div[1]/div[3]/form/div[1]/div[1]/div[2]/div[4]/div[6]/center/input[2]");
+  public static final By RESULT_HEADERS = By.xpath("//a/h3");
+}
```

Comment:
I would probably put your methods to interact with these elements, in this class also and then call those methods from your step defs class.
E.g. 
```
WebElement searchButton = wait.until(ExpectedConditions.elementToBeClickable(SearchPage.SEARCH_BUTTON));
    searchButton.click();
```

Reply:
Added all the methods to a class and now calling under Step defs